partner_identification/models/res_partner_id_category.py

```diff
@@ -58,14 +57,11 @@ def validate_id_number(self, id_number):
         try:
             safe_eval(self.validation_code, eval_context, mode="exec", nocopy=True)
         except Exception as e:
-            raise UserError(
-                _(
-                    "Error when evaluating the id_category validation code:"
-                    ":\n %s \n(%s)"
-                )
-                % (self.name, e)
-            )
+            msg = (
```

Comment:
@Freni-OSI @dreispt This is not good !

You should format after the translation!

Reply:
Can you check and review it again? Thanks